### PR TITLE
Solo anc stix ephemeris

### PR DESCRIPTION
**Pull request by nicHoch**
rename L2-aux files to ANC soar schema 

https://github.com/i4Ds/STIXCore/issues/396

### CODE REVIEW

File: stixcore/processing/publish.py [hunk 1]
```diff
@@ -539,12 +539,12 @@ def publish_fits_to_esa(args):
...
     parser.add_argument("-l", "--include_levels",
                         help="what levels should be published", type=str,
-                        default=CONFIG.get('Publish', 'include_levels', fallback="L0, L1, L2"))
+                        default=CONFIG.get('Publish', 'include_levels', fallback="L0, L1, L2, ANC"))
 
     parser.add_argument("-p", "--include_products",
                         help="what products should be published", type=str,
                         default=CONFIG.get('Publish', 'include_products',
-                                           fallback="ql,hk,sci,aux,cal"))
+                                           fallback="ql,hk,sci,asp,cal"))
```
Comment (samaloney) [inline stixcore/processing/publish.py:547]: asp = aspect or should it be anc = ancialliar?

**Review by samaloney — COMMENTED** [review]
LGTM would need to be updated once #393 is merged

File: stixcore/processing/publish.py (hunk 1 above)
Comment (nicHoch) [reply stixcore/processing/publish.py:547]: yes: asp = aspect 
ANC is allready in the level
and i added a new type as it is not HK anymore

**Review by samaloney — APPROVED** [review]
LGTM

**Pull request merged**